data/tribes/immovables/trees/balsa/terrain_affinity.lua

```diff
@@ -0,0 +1,74 @@
+terrain_affinity = {
+   preferred_temperature = 90,
+   preferred_humidity = 600,
+   preferred_fertility = 650,
+   pickiness = 35,
+}
+
+terrain_affinity_black = {
+   preferred_temperature = 100,
+   preferred_humidity = 150,
+   preferred_fertility = 850,
+   pickiness = 50,
+}
+
+terrain_affinity_desert = {
+   preferred_temperature = 140,
+   preferred_humidity = 500,
+   preferred_fertility = 500,
+   pickiness = 50,
+}
+
+terrain_affinity_winter = {
+   preferred_temperature = 45,
+   preferred_humidity = 750,
+   preferred_fertility = 400,
+   pickiness = 50,
+}
+
+spritesheet_sapling = { idle = {
```

Comment:
I think this should be in its own includable lua file. as it is not obvious to find this information in the terrain affinity file. 
same foar all amzon trees